app/context/toastContext.tsx

```diff
@@ -0,0 +1,32 @@
+import { createContext, useState, type ReactNode } from "react"
+
+export type ToastStatus = "success" | "failed" | "sent"
+
+type Toast = {
+  message: string | null
+  status: ToastStatus | null
```

Comment:
nit: In `~typing` we have `Nullable` type

```typescript
type Toast = {
    message: Nullable<string>
    status: Nullable<ToastStatus>
}
```

Reply:
[use Nullable in our codebase](https://github.com/pilagod/waallet-extension/pull/158/commits/5af4dd0a7e4636e9bf01f8d9aeec36bc8df4da41)